projects/02-meme-generator-api/implementation/app/controllers/application_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'sinatra/base'
+require './app/models/meme'
+
+class ApplicationController < Sinatra::Base
+  configure do
+    set :views, 'app/views'
+  end
+
+  get '/' do
+    erb :index
+  end
+
+  get '/memes' do
+    erb :memes_form
+  end
+
+  enable :sessions
```

Comment:
Since we changed the implementation not to use session, we can remove this.

The intro and first 2 sections in this article are relevant for this: https://www.justinweiss.com/articles/how-rails-sessions-work/
What is a session?
What does this have to do with a session?